filament/backend/src/noop/NoopDriver.cpp

```diff
@@ -31,7 +31,7 @@ NoopDriver::NoopDriver() noexcept : DriverBase(new ConcreteDispatcher<NoopDriver
 NoopDriver::~NoopDriver() noexcept = default;
 
 backend::ShaderModel NoopDriver::getShaderModel() const noexcept {
-#if defined(ANDROID) || defined(IOS)
+#if defined(ANDROID) || defined(IOS) || defined(EMSCRIPTEN)
```

Comment:
Everywhere else we use:
`#if defined(__EMSCRIPTEN__)`

@prideout can you confirm?

Reply:
Yes good catch Romain, let's be consistent with the rest of Filament.

Reply:
that's how I had it originally but I figured we should match ANDROID and IOS. It does work FWIW.

Reply:
The underscored version is more correct as it allows for emscripten's "strict" mode to be enabled, and is consistent with the rest of Filament.

Reply:
Done.